src/data/sketches.ts

```diff
@@ -17,6 +18,14 @@ export interface SketchMetadata {
 }
 
 export const SKETCHES: SketchMetadata[] = [
```

Comment:
Let's make a separate list of lab sketches for prototyping, like I did with `p5-sketchbook` earlier today